apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/SubscriberServiceApiClient.cs

```diff
@@ -926,14 +926,15 @@ public virtual Subscriber.SubscriberClient GrpcClient
 
         /// <summary>
         /// Creates a subscription to a given topic. See the
-        /// &lt;a href="/pubsub/docs/admin#resource_names"&gt; resource name rules&lt;/a&gt;.
+        /// &lt;a href="https://cloud.google.com/pubsub/docs/admin#resource_names"&gt;
```

Comment:
Flagging this file as containing more than just the copyright year change. It seems to be just documentation changes so it might be OK though.

Reply:
Well spotted! Will revert that part - I do want to keep the changes limited to just the copyright.